crates/chat-cli/src/cli/chat/prompt.rs

```diff
@@ -306,28 +326,10 @@ pub fn rl(
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-
-    #[test]
-    fn test_generate_prompt() {
```

Comment:
Moved these tests a bit lower and they test `highlight_prompt` now rather than just the prompt generation.